modules/develop/pages/kafka-clients.adoc

```diff
@@ -49,7 +49,7 @@ supported, reach out to the Redpanda team in the community https://redpanda.com/
 
 Redpanda does not currently support the following Apache Kafka features:
 
-* Managing SASL users with Kafka APIs: DescribeUserSCRAMCredentialsRequest and AlterUserSCRAMCredentialsRequest APIs.
+* Multiple SCRAM mechanisms simultaneously for SASL users, for example, a user having both a `SCRAM-SHA-256` and a `SCRAM-SHA-512` credential. Redpanda supports only one SASL/SCRAM mechanism per user, either `SCRAM-SHA-256` or `SCRAM-SHA-512`. See the xref:manage:security/authentication.adoc#sasl[Configure Authentication] guide for details. 
```

Comment:
Per the note here: https://redpandadata.atlassian.net/browse/ENG-306?focusedCommentId=62967